dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -316,6 +316,8 @@ public static JsonNode dockstoreEntryToElasticSearchObject(final Entry entry) th
         Set<String> verifiedPlatforms = getVerifiedPlatforms(workflowVersions);
         Entry detachedEntry = removeIrrelevantProperties(entry);
         JsonNode jsonNode = MAPPER.readTree(MAPPER.writeValueAsString(detachedEntry));
+        // add number of starred users to allow sorting in the UI
+        ((ObjectNode)jsonNode).put("stars_count", (long) entry.getStarredUsers().size());
```

Comment:
I think this means we'll have to reindex after we deploy. If so, I started a [hotfix checklist](https://github.com/dockstore/dockstore-deploy/wiki/1.12.3-Hotfix-Checklist) that should be updated.

Probably should add it to the 1.13 checklist as well, although it sounds like we might reindex it anyway.